tests/test_cli.py

```diff
@@ -55,11 +57,13 @@ def test_ls(mocker, capsys):
     assert isinstance(out, str)
 
 
-def test_arg_debug(mocker):
+def test_arg_debug(mocker: MockFixture):
     testargs = ["cz", "--debug", "info"]
     mocker.patch.object(sys, "argv", testargs)
     cli.main()
-    assert sys.excepthook.keywords.get("debug") is True
+    excepthook = sys.excepthook
+    assert isinstance(excepthook, partial)
```

Comment:
I'm a bit confused. Why is excepthook a partial?

Reply:
@Lee-W Because `sys.excepthook` is replaced by a `partial` here.
```py
    if args.debug:
        logging.getLogger("commitizen").setLevel(logging.DEBUG)
        sys.excepthook = commitizen_debug_excepthook
    elif args.no_raise:
        no_raise_exit_codes = parse_no_raise(args.no_raise)
        no_raise_debug_excepthook = partial(
            commitizen_excepthook, no_raise=no_raise_exit_codes
        )
        sys.excepthook = no_raise_debug_excepthook
```
https://github.com/commitizen-tools/commitizen/blob/master/commitizen/cli.py#L379-L387

and `keywords` is a property of `partial` object.

Maybe I should add some comments to explain this?

Reply:
ah got it. Yep, it would be really helpful! Even though it was what I implemented, I still forget this 🤦‍♂️